src/shared/components/Orgs/OrgList.tsx

```diff
@@ -1,83 +1,152 @@
 import * as React from 'react';
-import { Input, Empty } from 'antd';
-import OrgItem, { OrgItemProps } from './OrgItem';
-import AnimatedList from '../Animations/AnimatedList';
-
+import { Spin } from 'antd';
+import { OrgResponseCommon, OrganizationList } from '@bbp/nexus-sdk';
+import { PaginatedList } from '../../utils/types';
+import Search from 'antd/lib/input/Search';
+import OrgItem from './OrgItem';
+import { AccessControl, useNexus } from '@bbp/react-nexus';
 import './Orgs.less';
-import { Organization } from '@bbp/nexus-sdk-legacy';
+import { DEFAULT_UI_SETTINGS } from '../../utils/consts';
+import InfiniteScroll from '../Animations/InfiniteScroll';
+import { FetchableState } from '../../store/reducers/utils';
 
-export interface OrgListProps {
-  orgs: OrgItemProps[];
-  busy?: boolean;
-  error?: { message: string; name: string };
-  onOrgClick?(org: Organization): void;
-  onOrgEdit?(label: string): void;
-  paginationSettings?: { total: number; from: number; pageSize: number };
-  onPaginationChange?: (page: number, pageSize?: number) => void;
+export interface OrgListContainerProps {
+  pageSize?: number;
+  onOrgClick?(orgLabel: string): void;
+  onOrgEdit?(orgLabel: string): void;
+  createOrg?(): void;
 }
 
-const Search = Input.Search;
+export const OrgsListContainer: React.FunctionComponent<
+  OrgListContainerProps
+> = props => {
+  const {
+    pageSize = DEFAULT_UI_SETTINGS.pageSizes.orgsListPageSize,
+    onOrgClick,
+    onOrgEdit,
+    createOrg,
+  } = props;
+  const [{ from, size, label, deprecated }, setQuery] = React.useState({
+    size: pageSize,
+    from: 0,
+    label: '',
+    deprecated: false,
+  });
+  const [fetchablePaginatedList, setPaginatedList] = React.useState<
+    FetchableState<PaginatedList<OrgResponseCommon>>
```

Comment:
I'm not sure I get that type. Why do we need a `FetchableState of PaginatedList of OrgResponseCommon`. What about `OrganisationList` directly?

Reply:
The InfiniteScroll component requires it in this format, it was the standard until we changed the SDK so it made sense at the time.